app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+
+import { Button, Pagination, Table } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import { TABLE_ROW_DATA } from "./constants";
+import Menu from "./Menu";
+import { tableColumnData } from "./utils";
+
+const Contacts = () => {
+  const [searchTerm, setSearchTerm] = useState("");
+  const [openMenuBar, setOpenMenuBar] = useState(false);
```

Comment:
Prefix boolean variables with `is`, or `has`, or `should` etc, since it makes it sound like a question.

`openMenuBar` makes it sound like an action. We define actions as functions usually. But here it's not a function.

Thus a better name would be `isMenuBarOpen`.